### PR TITLE
Kamyroll -> Consumyroll

### LINKED ISSUE #1180: Kamyroll: problem loading video
### Source information and language

Kamyroll 13.7 (all)

### Steps to reproduce

When I try to play an anime it keeps loading in a infinite loop 

### Expected behavior

The video should play normally

### Actual behavior

But the video kept loading and nothing

### Aniyomi version

0.14.2.0-5863 (standard, 45d2af017, 93, 2023-01-07T02:26Z)

### Android version

Android 9 (SDK 28)

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.5](https://github.com/jmir1/aniyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://aniyomi.jmir.xyz/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/jmir1/aniyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #1181: [Bug] Kamyroll: video loading in infinite loop
The kamyroll has a problem, where the video is loading non-stop, the name of the anime and the episode does not appear, this problem only happened in dubbed animes.

### LINKED ISSUE #1210: [Bug] <kamyroll>: <failed conect>
This text is displayed when I enter the extension

### PR DESCRIPTION
**Pull request by Samfun75**
Remove Kamyroll and add consumet crunchyroll api as consumyroll (rename if you want)

closes #1210, closes #1181, closes #1180

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by jmir1** [comment]
note this: https://github.com/jmir1/aniyomi-extensions/blob/master/CONTRIBUTING.md#renaming-existing-sources
and one question: do old kamyroll anime work in your aniyomi library with the new "consumyroll" extension, or do you have to migrate?

**Comment by Samfun75** [comment]
At first I renamed it, but then since it was a completely different source I thought to make it its own extension. And since the info about the audio is not provided with the stream like kamy, that info is saved in the episode url so they have to be refreshed. So they don't necessarily have to migrate but refresh the episode list for all the kamy anime in their library if renamed. That's one is the reasons I made it is own extension. So they could mass migrate it. 

Should I still rename it?

**Comment by jmir1** [comment]
Now you changed it to renaming the extension. I don't have an opinion on which is better. Just do what is best in your opinion! :+1:

**Comment by Samfun75** [comment]
Yeah renaming it allows refreshing episode list and migration so I think it's fine to just rename it. I'm trying to change the epsiode list to use Crunchyroll instead of consumet to lessen the strain on their servers. I already finished it but forgot to push it before I left my computer. I'll push it tomorrow

File: src/all/kamyroll/src/eu/kanade/tachiyomi/animeextension/all/kamyroll/Consumyroll.kt [hunk 1]
```diff
@@ -62,45 +67,40 @@ class Kamyroll : ConfigurableAnimeSource, AnimeHttpSource() {
 
     // ============================== Popular ===============================
 
-    override fun popularAnimeRequest(page: Int): Request =
-        GET("$baseUrl/content/v1/updated?channel_id=$channelId&limit=20")
+    override fun popularAnimeRequest(page: Int): Request {
+        val start = if (page != 1) "start${(page - 1) * 36}&" else ""
```
Comment (Secozzi) [inline src/all/kamyroll/src/eu/kanade/tachiyomi/animeextension/all/kamyroll/Consumyroll.kt:71]: You seem to have forgotten to add an `=` after `start`

Comment (Samfun75) [reply src/all/kamyroll/src/eu/kanade/tachiyomi/animeextension/all/kamyroll/Consumyroll.kt:71]: Haha it seems I haven't scrolled pass page 1 in my testing, thanks for catching that

**Pull request merged**